src/main/java/jarvis/display/DisplayMessages.java

```diff
@@ -0,0 +1,136 @@
+
+        System.out.println(message);
+    }
+
+    public static void closingMessage() {
+        String message = HORIZONTAL_LINE
+                + "______            _ \n"
+                + "| ___ \\          | |\n"
+                + "| |_/ /_   _  ___| |\n"
+                + "| ___ \\ | | |/ _ \\ |\n"
+                + "| |_/ / |_| |  __/_|\n"
+                + "\\____/ \\__, |\\___(_)\n"
+                + "        __/ |       \n"
+                + "       |___/        \n"
+                + "______                      _                   _                           \n"
+                + "| ___ \\                    (_)                 | |                          \n"
+                + "| |_/ /____      _____ _ __ _ _ __   __ _    __| | _____      ___ __        \n"
+                + "|  __/ _ \\ \\ /\\ / / _ \\ '__| | '_ \\ / _` |  / _` |/ _ \\ \\ /\\ / / '_ \\       \n"
+                + "| | | (_) \\ V  V /  __/ |  | | | | | (_| | | (_| | (_) \\ V  V /| | | |_ _ _ \n"
+                + "\\_|  \\___/ \\_/\\_/ \\___|_|  |_|_| |_|\\__, |  \\__,_|\\___/ \\_/\\_/ |_| |_(_|_|_)\n"
+                + "                                     __/ |                                  \n"
+                + "                                    |___/                                   \n"
+                + HORIZONTAL_LINE;
+        System.out.println(message);
+    }
+
+    public static void emptyList() {
+        String message = HORIZONTAL_LINE
+                    + "Your list is currently empty. Try adding some items to your list first.\n"
+                    + HORIZONTAL_LINE;
+        System.out.println(message);
+    }
+
+    public static void taskAdded(String taskDescription) {
+        String message = HORIZONTAL_LINE
+                + "added: " + taskDescription + "\n"
+                + HORIZONTAL_LINE;
+        System.out.println(message);
+    }
+
+    public static void markError() {
+        System.out.print(HORIZONTAL_LINE);
+        System.out.println("This task is already marked as not done! Did you mean to mark it?");
+        System.out.print(HORIZONTAL_LINE);
+    }
+
+    public static void unmarkError() {
+        System.out.print(HORIZONTAL_LINE);
+        System.out.println("This task is already marked as done! Did to mean to unmark it?");
+        System.out.print(HORIZONTAL_LINE);
+    }
+
+    public static void invalidInput() {
+        System.out.println(HORIZONTAL_LINE
+                + "You have entered an invalid command. Please try again\n"
+                + HORIZONTAL_LINE);
+    }
+
+
+    public static void taskDeleted(Task taskRemoved, Integer listSize) {
+        System.out.print(HORIZONTAL_LINE + "I have removed this task for you:\n" + taskRemoved.getFullTask() + "\n"
+                + "You have " + listSize.toString() + " tasks left.\n" +
+                HORIZONTAL_LINE);
+    }
+
+        public static void savedFileDetected () {
```

Comment:
**Coding standard violation:** There should not be any whitespace between `savedFileDetected` and `()`